docker/dockerfile-build-env/latest/Dockerfile

```diff
@@ -22,35 +21,15 @@ RUN apt-install \
     && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - \
     && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu    $(lsb_release -cs)    stable" \
     && apt-install docker-ce \
+    && usermod -aG docker application \
     && pip install --upgrade pip \
     && hash -r pip \
     && pip install --upgrade setuptools \
-    && pip install --upgrade \
-        MarkupSafe \
-        Jinja2 \
-        graphviz \
-        funcsigs \
-        pbr \
-        six \
-        mock \
-        psutil \
-        pylev \
-        py \
-        appdirs \
-        pyparsing \
-        packaging \
-        pytest \
-        simplejson \
-        backpack \
-        cleo \
-        pyyaml \
-        yamlordereddictloader \
-        testinfra \
-        cloudpickle \
-        pyinotify \
-        configparser \
-        doit==0.29.0 \
-        termcolor \
-        pytest-timeout \
-        pytest-rerunfailures \
+    && git clone https://github.com/webdevops/Dockerfile.git /tmp/Dockerfile \
```

Comment:
You should use `--depth 1` to clone only the latest master version (reduce size drastically and speedup)